python/lsst/afw/geom/angle/angle.cc

```diff
@@ -90,20 +90,22 @@ PYBIND11_PLUGIN(angle) {
                  py::is_operator());
     clsAngle.def("__sub__", [](Angle const& self, Angle const& other) { return self - other; },
                  py::is_operator());
+    clsAngle.def("__neg__", [](Angle const& self) { return -self; }, py::is_operator());
     clsAngle.def("__iadd__", [](Angle& self, Angle const& other) { return self += other; });
     clsAngle.def("__isub__", [](Angle& self, Angle const& other) { return self -= other; });
     clsAngle.def("__truediv__", [](Angle const& self, double other) { return self / other; },
                  py::is_operator());
+    // Without an explicit wrapper, Python lets Angle / Angle -> Angle
+    clsAngle.def("__truediv__", [](Angle const& self, Angle const& other) {
+        throw py::type_error("unsupported operand type(s) for /: 'Angle' and 'Angle'");
+    });
 
     clsAngle.def("__float__", &Angle::operator double);
```

Comment:
Can we get rid of this? I suppose it may require an RFC (though if it passed Jenkins...)